examples

```diff
@@ -0,0 +1 @@
+packages/nibijs-examples
```

Comment:
What is this no extension file doing?

Reply:
Doesn't work on windows :(

Reply:
How do we do Windows symlinks?

Reply:
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/mklink

Reply:
mklink /D examples packages\nibijs-examples

Worked for me

Reply:
However I think this is only locally applied

Reply:
Hmm, there may be a way to make it respect both symlinks